<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>668</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>833</width>
    <height>668</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title_Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>721</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> Warning Isolator v2.08</string>
    </property>
   </widget>
   <widget class="QLabel" name="logpath_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>File Path:</string>
    </property>
   </widget>
   <widget class="QLabel" name="compiler_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Compiler:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="complierselect_comboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="compiler_Label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save Path:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gen_pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS UI Gothic</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="log_textEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Please give the path of the log file...</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="save_textEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>711</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Where do you want to save it???</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_pushButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>350</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS UI Gothic</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="update_textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>811</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QRadioButton" name="diab_warning_radioButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Warning</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="diab_info_radioButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>   Info</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="diab_boith_radioButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>   Both</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
